tests/appsec/waf/test_reports.py

```diff
@@ -16,6 +16,11 @@ class Test_Monitoring:
     def setup_waf_monitoring(self):
         self.r = weblog.get("/waf/", headers={"User-Agent": "Arachni/v1"})
 
+    @flaky(
+        library="java",
+        weblog_variant="vertx4",
+        reason="Although span tags are correct some times rasp is not blocking du to timing issue",
+    )
```

Comment:
If a JIRA ticket exists, just set its ID, it'll easier to track, and retrieve.

```suggestion
    @flaky(
        library="java",
        weblog_variant="vertx4",
        reason="APPSEC-54465",
    )
```

Reply:
Thanks!